config/initializers/node-orm2.js

```diff
@@ -18,12 +18,15 @@ export default {
       }).then((db) => {
         adapter.db = db;
         let models = container.lookupAll('model');
-        let adapterModels = mapValues(models, (Model) => {
+
+        let adapterModels = {};
+        forEach(models, (Model, key) => {
```

Comment:
I changed from `mapValues` to `forEach` here because `mapValues` was leaving an `undefined` value in the `adapterModels` hash which then broke `adapter.defineRelationships` below ...